routers/repo/download.go

```diff
@@ -46,6 +46,11 @@ func ServeData(ctx *context.Context, name string, reader io.Reader) error {
 	} else if base.IsImageFile(buf) || base.IsPDFFile(buf) {
 		ctx.Resp.Header().Set("Content-Disposition", fmt.Sprintf(`inline; filename="%s"`, name))
 		ctx.Resp.Header().Set("Access-Control-Expose-Headers", "Content-Disposition")
+		if base.IsSVGImageFile(buf) {
+			ctx.Resp.Header().Set("Content-Security-Policy", "default-src 'none'; style-src 'unsafe-inline'; sandbox")
```

Comment:
Is `unsafe-inline` necessary here? ref. https://content-security-policy.com/unsafe-inline/

Reply:
Without it, many SVG images look nothing like they're intended to look.  Many SVG attributes have a style alternative.

That said, I originally omitted it, but I was asked to add it in another comment in this PR.  As long as it's safe, I think that including it is a good thing.

Reply:
Please see my ref article, It said
>
When you want to allow inline scripts or styles on a page that uses CSP, there two much better options: nonce or hash.

Reply:
I read the article.  I'm having trouble understanding how nonce or hash is actually better.  If I understand it correctly, a hash prevents modification between the server and client, which is not more than what HTTPS would do.  So this is to make the HTTP case safer.

If you think that's what we should do, it's not clear to me how to actually implement it.  It seems like we would need to parse the SVG file enough to either insert nonces at the appropriate places in the SVG or read all the inline styles so that we can compute their hashes.  Is this what you're saying?  I'm guessing this must include all the `style="..."` attributes.  There could potentially be many of them.  I'll think about it some more.

Reply:
@lunny The suggestion from the ref article is not applicable in this case, as the CSP is applied when rendering user-generated content (the SVG file). The general purpose of using nonce is so that website can include inline script in non-user controlled HTML content while at the same time benefiting from CSP blocking possible XSS attacks.

Reply:
nonce of CSP requires a change to the svg content dynamiclly which will make the thing complicated.